src/common/src/config.rs

```diff
@@ -982,6 +982,69 @@ pub struct ObjectStoreConfig {
     #[serde(default = "default::object_store_config::object_store_set_atomic_write_dir")]
     pub object_store_set_atomic_write_dir: bool,
 
+    #[serde(default = "default::object_store_config::object_store_req_retry_interval_ms")]
+    pub object_store_req_retry_interval_ms: u64,
```

Comment:
Collect all the retry and timeout related config into a separated config struct?